clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -229,7 +264,6 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
         if (Method->isVirtual())
           SemaRef.Diag(e->getExprLoc(), diag::err_sycl_restrict)
               << Sema::KernelCallVirtualFunction;
-
```

Comment:
Are you removing a bunch of lines in here?  Why? (next 3 changes).

Reply:
Please don't.